client/packages/invoices/src/StockOut/utils.tsx

```diff
@@ -221,6 +224,8 @@ export const allocateQuantities =
       ...batch,
       numberOfPacks: 0,
       isUpdated: batch.numberOfPacks > 0,
+      prescribedQuantity:
+        batch.numberOfPacks > 0 && prescribedQuantity ? prescribedQuantity : 0,
```

Comment:
This is a temporary solution. The main goal is to only add prescribedQuantity to one line. But we've decided to tackle that in a different ticket for ease of review (this one is already big enough sorry 🙏🏽)

[Improvements Ticket](https://github.com/orgs/msupply-foundation/projects/21/views/7?pane=issue&itemId=95764653&issue=msupply-foundation%7Copen-msupply%7C6353)

Reply:
Potential solution for the above: https://github.com/msupply-foundation/open-msupply/commit/164f6cba1e4d754012ca980a9f5ff2402a1e2bdf

Reply:
PR for the ticket mentioned in the thread: https://github.com/msupply-foundation/open-msupply/pull/6387

Reply:
6387 is merged to this branch